README.md

```diff
@@ -39,6 +39,7 @@ It must contain at least these attributes:
 ```ruby
 json_bill = {
   cae: "1234567890123",       # CAE number
+  cae_due_date: "20171125",   # CAE expiry date
```

Comment:
@tcordeu Could you point us in the right direction here? I haven't been keeping up with the latest AFIP API changes. Do you have a screenshot, sample, or link to official documentation? Trying to understand if this is a new field, or the same field (`fch_vto_pago`) being renamed. 

If you are adding `cae_due_date` here, shouldn't you remove `fch_vto_pago` from the README?

Reply:
@tcordeu As @etagwerker said, we should get rid of `fch_vto_pago` from the README.